lib/resources/example_json/dstu2_examples_contract.rb

```diff
@@ -0,0 +1,613 @@
+            },
+            "applies": {
+              "start": "2016-10-18T17:32:55.000Z"
+            },
+            "subject": [
+              {
+                "reference": "Patient/14067775",
+                "display": "NOAH, SALLY"
+              }
+            ],
+            "type": {
+              "coding": [
+                {
+                  "system": "https://snomed.ct/sct",
+                  "code": "371537001",
+                  "display": "Consent report (record artifact)"
+                }
+              ],
+              "text": "Consent report (record artifact)"
+            },
+            "subtype": [
+              {
+                "coding": [
+                  {
+                    "system": "https://snomed.ct/sct",
+                    "code": "309370004",
+                    "display": "Consent status (finding)"
+                  }
+                ],
+                "text": "Consent status (finding)"
+              }
+            ],
+            "action": [
+              {
+                "coding": [
+                  {
+                    "system": "https://snomed.ct/sct",
+                    "code": "441898007",
+                    "display": "Consented (qualifier value)"
+                  }
+                ],
+                "text": "Consented (qualifier value)"
+              }
+            ],
+            "actionReason": [
+              {
+                "coding": [
+                  {
+                    "system": "https://snomed.ct/sct",
+                    "code": "425691002",
+                    "display": "Consent given for electronic record sharing (finding)"
+                  }
+                ],
+                "text": "Consent given for electronic record sharing (finding)"
+              }
+            ],
+            "actor": [
+              {
+                "entity": {
+                  "reference": "RelatedPerson/11365084",
+                  "display": "NOAH, SANDRA"
+                },
+                "role": [
+                  {
+                    "coding": [
+                      {
+                        "system": "http://snomed.info/sct",
+                        "code": "442681007",
+                        "display": "Recipient consented (qualifier value)"
+                      }
+                    ],
+                    "text": "Authorized Representative"
+                  }
+                ]
+              }
+            ]
+          }
+        }
+      ]
+    }
+
+    DSTU2_CONTRACT_BUNDLE_INCLUDE_ACTOR ||= {
+      "resourceType": "Bundle",
+        "id": "bae9d7ad-79e4-4c85-9899-ef93f85b2dd5",
+        "type": "searchset",
+        "total": 2,
```

Comment:
Yes. The total represents the number of search results and not the number of entries. There are 2 Contracts returned